src/translations/screens/subscreens/MobilityTexts.ts

```diff
@@ -88,6 +88,11 @@ export const MobilityTexts = {
     'Inkludert i billetten',
   ),
   showInMap: _('Se i kart', 'Show in map', 'Sjå i kart'),
+  showInMapA11yLabel: _(
+    'Aktiver for å se i kart',
```

Comment:
The map is disabled when screenreader is activated, so this won't work?

Reply:
Correct! We (me, @oyvindmikkelsenatb, @HanneLH ++) had a discussion about possible solutions to this yesterday, and we landed on this as a solution that's as close to good as we'll get for now 😅 Please let me know if you have better ideas.

Reply:
Hmm, thinking about it now - if you can't see the screen, you shouldn't drive or ride a bike either. So with the current lineup of benefits it shouldn't matter.

Reply:
I'm sure renting a car can be useful, since someone else can drive it. Would be nice if we could redirect directly to Hyre (or any other provider app) instead.

Reply:
Yea. Ideally you shouldn't have to go through the map at all to activate the benefits, so that would be the best solution.
However if you have someone to drive for you, maybe you also have someone to help you use the benefit in the app?
Another potential issue is if the 3rd party app doesn't support screenreader, but nothing we can do about that..

Reply:
Agreed :+1: